pre-processing/primod/mapping/rch_svat_mapping.py

```diff
@@ -39,12 +39,13 @@ def __init__(self, svat: xr.DataArray, recharge: mf6.Recharge):
         super().__init__()
         self.dataset["svat"] = svat
         self.dataset["layer"] = xr.full_like(svat, 1)
-        if "layer" in recharge.dataset.coords:
-            self.dataset["rch_active"] = (
-                recharge.dataset["rate"].drop_vars("layer").notnull()
-            )
+        rate = recharge.dataset["rate"]
+        if "layer" in rate.dims:
+            rate_no_layer = rate.squeeze("layer", drop=True)
```

Comment:
should there be a check that the length of the layer dimension is 1?

Reply:
The squeeze method automatically throws an error if length is more than 1.